### PR TITLE
chore: configure release drafter

### PR DESCRIPTION
**Pull request by ennru**

### CODE REVIEW

**Review by andreaTP — APPROVED** [review]

**Comment by andreaTP** [comment]
Thanks!

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Config for https://github.com/release-drafter/release-drafter
+name-template: 'Akka Serverless Javascript/Typescript SDK $NEXT_PATCH_VERSION'
+tag-template: 'v$NEXT_PATCH_VERSION'
+categories:
+  - title: '🚀 Productboard Features'
```
Comment (pvlugter) [inline .github/release-drafter.yml:5]: Should it just be `Features` (without Productboard) for the OSS repos?

File: .github/release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+autolabeler:
+  - label: 'chore'
+    title:
+      - '/chore:/i'
+  - label: 'Documentation'
+    files:
+      - 'docs/*'
+  - label: 'tck'
+    files:
+      - 'tck/*'
```
Comment (pvlugter) [inline .github/release-drafter.yml:38]: Should we add other conventional commit types that we'll be using, maybe `fix` in particular for a fixes section?

**Review by pvlugter — APPROVED** [review]
👍🏼

**Pull request merged**